public/wp-content/themes/newsup-child/single-anime.php

```diff
@@ -88,10 +88,10 @@
                         </div>
                         <div class="media-body">
                             <h2>Author</h2>
-                            <p><b><?php echo $author; ?></b></p>
+                            <p><b><?php if($author) echo $author; ?></b></p>
                             <hr>
                             <h2>Release Date</h2>
-                            <p><b><?php echo $release_date; ?></b></p>
+                            <p><b><?php if($release_date) echo $release_date; ?></b></p>
```

Comment:
If it have no author or date you going to just leave it blank?

Reply:
good eye, ill refactor this xD tyyy

Reply:
gateeeem